doc/development.md

```diff
@@ -26,7 +26,7 @@ Clone your fork:
 
 Install additional packages for Fedora, CentOS, and RHEL:
 
-    sudo dnf install -y `cat ci/check-patch.packages`
+    contrib/install-pkg.sh
```

Comment:
contrib/install-pkg.sh is not the best name, and the script does have any documentation so someone looking in this script will have no idea what is the purpose of it.

Reply:
Good idea: https://github.com/oVirt/vdsm/issues/309

I can tackle this in a followup PR so that we unblock the SPDX branch.